tests/local_contract_calls.py

```diff
@@ -95,8 +106,18 @@ class ReceiptAggregateVoucher(EIP712Message):
 
 # MOCK STAKING CONTRACT CALLS
 try:
+    message_hash = Web3.solidity_keccak(
+        ["address", "address"], [GATEWAY, ALLOCATION_ID]
+    )
+
+    # Convert the hash to the Ethereum specific signature format
+    eth_signed_message = encode_defunct(hexstr=message_hash.hex())
+
+    # Sign the message with the private key
+    signature = w3.eth.account.sign_message(eth_signed_message, private_key=ALLOCATIONID_PK)
+    arbitraryBytes32 = os.urandom(32)
     print("Allocate receiver with allocationid")
-    mockStaking.functions.allocate(ALLOCATION_ID, RECEIVER).transact({"from": GATEWAY})
+    mockStaking.functions.allocate(arbitraryBytes32, 1000, ALLOCATION_ID, arbitraryBytes32, RECEIVER).transact({"from": RECEIVER})
```

Comment:
This is OK, but ideally should be in another PR, since it's not about supporting `manyDeposits`.

Reply:
Nah, we'll keep it here, but it'd be better for next times.